app/hocs/withCopyCanvasToClipboard.js

```diff
@@ -45,7 +45,7 @@ export default function withCopyCanvasToClipboard(Component) {
       // $FlowFixMe
       const url = ref.toDataURL('image/jpg', 0.8)
       const base64Data = url.replace(/^data:image\/png;base64,/, '')
-      const storagePath = storage.getDefaultDataPath()
+      const storagePath = storage.getDataPath()
```

Comment:
I am pretty sure this is not going to give you the path we use everywhere else in the application... We should be using `const path = await ipcRenderer.invoke('getStoragePath')` here

Reply:
All right. I was a bit confused on how to get the storage path. Thanks for the alert, I will fix this.